lib/Gitlab/Api/Issues.php

```diff
@@ -58,6 +58,16 @@ public function update($project_id, $issue_id, array $params)
      * @param int $issue_id
      * @return mixed
      */
+    public function remove($project_id, $issue_id)
```

Comment:
@brammeleman let's pls use `delete` on the method name, to keep the API's naming: https://docs.gitlab.com/ce/api/issues.html#delete-an-issue.